src/utils/utils_log.c

```diff
@@ -82,7 +82,7 @@ static void util_log_internal(util_log_level_t level, int perror,
     pid_t tid = (pid_t)tid64;
 #else
     pid_t pid = getpid();
-    pid_t tid = gettid();
+    pid_t tid = syscall(SYS_gettid);
```

Comment:
In the PR #505 I added `int utils_getpid(void);` function to the `utils_common.h`.

I missed this part of the code, but it would be great to move `gettid` implementation to the `utils_common.h` as well.
Would you be able to do it in this PR?

Reply:
Done